lib/ca_test.go

```diff
@@ -351,6 +352,98 @@ func TestCAgetUserAttrValue(t *testing.T) {
 	CAclean(ca, t)
 }
 
+func TestCAAddIdentity(t *testing.T) {
+
+	tests := []struct {
+		testName string
+		id       *CAConfigIdentity
+	}{
+		{
+			testName: "When identity has user and pass",
+			id: &CAConfigIdentity{
+				Name: "admin1",
+				Pass: "adminpw",
+			},
+		},
+		{
+			testName: "When identity has user and passfile",
+			id: &CAConfigIdentity{
+				Name:     "admin2",
+				PassFile: filepath.Join(os.TempDir(), "adminpwfile"),
+			},
+		},
+		{
+			testName: "When identity has user, pass, and passfile",
+			id: &CAConfigIdentity{
+				Name:     "admin3",
+				Pass:     "adminpw",
+				PassFile: filepath.Join(os.TempDir(), "adminpwfile"),
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
```

Comment:
Nit: this isn't necessary unless the tests are run with `t.Parallel()`, right?

Reply:
Correct, initially I tried to run these tests in parallel but this caused an issue because of the shared artifacts. 🧹